cycle/docker-compose.yaml

```diff
@@ -378,7 +378,7 @@ services:
         dds_community_build: ${dds_community_build_el7}
         dds_private_version: ${dds_private_version}
         dds_private_build: ${dds_private_build}
-        ts_dds: ${ts_dds_private}
+        ts_dds: ${ts_dds_private}=${ts_dds_private_conda_build}
```

Comment:
It that really an = sign or should it be a -?

Reply:
it is an "=" (https://stackoverflow.com/questions/48128029/installing-specific-build-of-an-anaconda-package)